emulator/src/Display.cpp

```diff
@@ -0,0 +1,53 @@
+#include "../include/Display.h"
+
+std::unique_ptr<FakeDisplay> fakeDisplayInstance;
+
+FakeDisplay::FakeDisplay(int width, int height, SDL_Window* window, SDL_Renderer* renderer) : Arduino_G(width, height), mainWindow(window), mainRenderer(renderer), width(width), height(height) { };
+
+FakeDisplay::~FakeDisplay() {
+
+};
```

Comment:
We should remove the trailing `;`in the source files. These are for heady only :)